Examples/Google/SpeechToText/Sources/AudioStreamManager.swift

```diff
@@ -0,0 +1,209 @@
+/*
+ * Copyright 2020, gRPC Authors All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// NOTE: Implementation based off of Google's for Audio Streaming:
+// https://github.com/GoogleCloudPlatform/ios-docs-samples/blob/master/speech/Swift/Speech-gRPC-Streaming/Speech/AudioController.swift
+
+import Foundation
+import AVFoundation
+
+enum AudioStreamError: Error {
+  case failedToConfigure
+  case failedToFindAudioComponent
+  case failedToFindMicrophoneUnit
+}
+
+protocol StreamDelegate: AnyObject {
+  func processAudio(_ data: Data)
+}
+
+class AudioStreamManager {
+  
+  var microphoneUnit: AudioComponentInstance?
+  weak var delegate: StreamDelegate?
+
+  static var shared = AudioStreamManager()
+
+  private let bus1: AudioUnitElement = 1
+  private var oneFlag: UInt32 = 1
```

Comment:
could that variable be local to the method where it is used? If not, document this variable and maybe the other ones as well? (Same goes for `bus1` above.)